3_javascript/javascript1/week1/A house price estimator.js

```diff
@@ -0,0 +1,25 @@
+// Peter
+let volumeInMetersPeter=8*10*10;
+let gardenSizeInM2Peter=100;
+let housePricePeter;
+housePricePeter = volumeInMetersPeter * 2.5 * 1000 + gardenSizeInM2Peter * 300;
+console.log(housePricePeter);
+if(housePricePeter>2.500000){
+ console.log("Peter's Pricing is high");
+}
+else{
+ console.log("Peter's Pricing is low"); 
+}
+
+// Julia
+let volumeInMetersJulia=5*11*8;
+let gardenSizeInM2Julia=70;
+let housePriceJulia;
+housePriceJulia = volumeInMetersJulia * 2.5 * 1000 + gardenSizeInM2Julia * 300;
+console.log(housePriceJulia);
+if(housePriceJulia> 1.000000){
+ console.log("Julia's Pricing is high");
+}
+else{
+ console.log("Julia's Pricing is low"); 
+}
```

Comment:
🚩 When comparing the prices, ensure you're using the correct values. Currently, you're comparing with `2.500000` which translates to 2.5. Use `2500000` instead.